app/controllers/fe/applications_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Fe
   class ApplicationsController < ApplicationController
```

Comment:
Why does this file exist in Fe? Looks like it contains logic specific to summer missions.